protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1381,15 +1381,15 @@ message WithdrawalsConnection {
 
 // -- Assets --
 message GetAssetRequest {
-    string asset_id = 1;
+    string id = 1;
```

Comment:
ditto

Reply:
Yes, this is the same as the reasoning in https://github.com/vegaprotocol/vega/pull/5895#discussion_r948796331
in the task it says:
`https://github.com/vegaprotocol/vega/pull/5895#discussion_r948796331`